docs/tools/kit/index.md

```diff
@@ -377,11 +377,12 @@ function RandomValues() {
 
 #### Notes:
 
-* Randomness is generated using Flow’s **on-chain `revertibleRandom`**, producing pseudorandom values tied to block and transaction execution.
-* The value returned for identical calls within the same block will be identical.
-* This hook is designed for simple use cases that **don’t require a [commit-reveal scheme]**, such as randomized UIs.
-* For **more advanced use cases** that **do** require a [commit-reveal scheme], Flow provides built-in support for that pattern.
-* Values are **deterministic**: if a transaction fails and retries, the same random value will be regenerated.
+* Randomness is generated using Flow’s **on-chain `revertibleRandom`**, producing pseudorandom values tied to block and script execution.
+* Values are **deterministic**: The values returned for identical calls within the same block will be identical.
+* If `count ` is larger than one, the returned values are disctinct.
+* This hook is designed for simple use cases that don't require unpredictability, such as randomized UIs.
```

Comment:
commit-reveal solves the reversibility issue, but here in the script nothing is reversible so commit-reveal doesn't add any advantage. 

The main thing developers should be aware of is that the block random source in the lib is publicly known (since this is script-based). There is nothing wrong with that in many applications like the example mentioned here (UIs)